### DISCUSSION TITLE
Spelling/Grammar/Dialog Improvement

### DISCUSSION THREAD

**Discussion by MonkeyMatrix** [OcelotWalrus/Cromha-Expansion-plugin#132] [Plugin Enhancment]
Ask any questions or propose changes to the spelling, grammar, or dialog present in the [Cromha Expansion plugin](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin). Please provide specific information about an error rather than generalizations (Unless it is of the same nature as changing "colonnies" to "colonies.").

**Comment by MonkeyMatrix** [comment]
<h1>Notable Changes</h1>

- Changed "colonnies" to "colonies" throughout the project.
- Changed "payed" to "paid" throughout the project.
- Changed "delivry" to "deliver" throughout the project.
- Changed "prisonner" to "prisoner" throughout the project.
- Changed "an other" to "another" throughout the project.

**Comment by MonkeyMatrix** [comment]
# Empire Questions

**<details><summary>Ships</summary>**
<p>

- [x] [Empire/empire bulkers.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/aa826ce410b70cbdf1acc909267705f234aa4e7e/data/Empire/empire%20bulkers.txt#L112): Do we classify outfits and ships as having models, or are ships the only ones classified as having models?

> The Mineria colonnies started attacking small Kor Efreti ships to collect outfits and ships models.

- [ ] [Empire/empire emperor warships.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/aa826ce410b70cbdf1acc909267705f234aa4e7e/data/Empire/empire%20emperor%20warships.txt#L353): What is "yard time?"

> Navarchs fight with the civilian government for resources and yard time they would rather devote to building...

</p>
</details>

---

**<details><summary>Jobs</summary>**
<p>

- [x] [Empire/empire jobs.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/main/data/Empire/empire%20jobs.txt): Should "credits" be added to the end of jobs after \<payment>?

> Payment is \<payment>.

- [ ] [Empire/empire jobs.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/main/data/Empire/empire%20jobs.txt#L310): What is a "sas?"

> After opening the ship's sas, the...

- [x] [Empire/empire jobs.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/main/data/Empire/empire%20jobs.txt#L2071): Should we remove "the date of?"

> Deliver a secure package to \<destination> before the date of \<date>.

</p>
</details>

---

**<details><summary>Outfits</summary>**
<p>

- [x] [Empire/empire outfits.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/main/data/Empire/empire%20outfits.txt#L466): Should "Tre-Phasic" be "Tri-Phasic?"

> outfit "Tre-Phasic Cooler"

- [ ] [Empire/empire outfits.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/main/data/Empire/empire%20outfits.txt#L476): Should the tre (or tri)-phasic cooler be given its own description?

- [x] [Empire/empire outfits.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/aa826ce410b70cbdf1acc909267705f234aa4e7e/data/Empire/empire%20power.txt#L144): Do these reactors produce 0 heat or almost 0 heat? I see a heat generation value of 8 and a cooling value of 3, but I am not entirely sure how the heat value is calculated in plugins.

> It has cooling ducts integrated to its systems, that way it produce literally no heat..

- [x] [Empire/empire prototypes outfits.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/aa826ce410b70cbdf1acc909267705f234aa4e7e/data/Empire/empire%20prototypes%20outfits.txt#L254): How should this be rephrased?

> It will also deal a noticeable amount of ion damage that can completely disable your ship energy long enough for you to disable them.

</p>
</details>

---

**<details><summary>Storyline</summary>**
<p>

- [ ] [Empire/empire storyline pug worker.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/f4618b48c1ce66836054ac51abc63930e9e11272/data/Empire/empire%20storyline%20pug%20worker.txt#L15): What is "arrimage?"

> You go back to your ship to do some things when you hear a noise near the ship arrimage.

</p>
</details>

---

#### Unchecked Box: Not Answered
#### Checked Box: Answered

**Reply by lumbar527** [reply]
I think credits might be included after `payment`.

Just to be clear, I mean that `<payment> credits` will become `Number credits credits`.

**Reply by lumbar527** [reply]
> [Empire/empire jobs.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/main/data/Empire/empire%20jobs.txt#L2071): Should we remove "the date of?"

Yes, I think so.

**Reply by MonkeyMatrix** [reply]
I will go look at that then, thank you for the insight.

**Reply by lumbar527** [reply]
> [Empire/empire jobs.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/main/data/Empire/empire%20jobs.txt#L310): What is a "sas?"

Uh, no idea. @OcelotWalrus?

**Reply by lumbar527** [reply]
> [Empire/empire bulkers.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/aa826ce410b70cbdf1acc909267705f234aa4e7e/data/Empire/empire%20bulkers.txt#L112): Do we classify outfits and ships as having models, or are ships the only ones classified as having models?

I would say both.

**Reply by lumbar527** [reply]
> [Empire/empire emperor warships.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/aa826ce410b70cbdf1acc909267705f234aa4e7e/data/Empire/empire%20emperor%20warships.txt#L353): What is "yard time?"

Time spent working maybe?

**Reply by lumbar527** [reply]
> [Empire/empire outfits.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/main/data/Empire/empire%20outfits.txt#L466): Should "Tre-Phasic" be "Tri-Phasic?"

I would say no; I don't see three parts in the image and there is also a Tre-Phasic Rifle image (didn't check for the outfit)

**Reply by MonkeyMatrix** [reply]
Yep, \<payment> does add credits. I will take that out of the notable changes and my local project.

**Reply by lumbar527** [reply]
> [Empire/empire outfits.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/main/data/Empire/empire%20outfits.txt#L476): Should the tre (or tri)-phasic cooler be given its own description?

Yeah, probably. I would have a separate discussion for description-things if there are enough of them.

**Reply by MonkeyMatrix** [reply]
I have been looking at descriptions for ships, outfits, etc. most of the time. Hopefully I did not mess with too much in this side-project. I think I will push the commits to my fork when I finish the Empire files.

**Reply by lumbar527** [reply]
> [Empire/empire outfits.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/aa826ce410b70cbdf1acc909267705f234aa4e7e/data/Empire/empire%20power.txt#L144): Do these reactors produce 0 heat or almost 0 heat? I see a heat generation value of 8 and a cooling value of 3, but I am not entirely sure how the heat value is calculated in plugins.

Looks like a little bit of heat generation:

<img width="300" alt="Screen Shot 2023-10-14 at 1 43 33 PM" src="https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/assets/125756761/e47e9e6e-ca19-4ad7-bfbd-6301c90a4a4a">

**Reply by MonkeyMatrix** [reply]
There are a few changes to other files in the project if they refer to events, ships, etc. from the Empire government.

**Comment by OcelotWalrus** [comment]
Hey! About the change of "colonnies" not "colonies", i totally refactored every occurenses of `Mineria colonnies` to `Mineria Colonies` through the entire repository by commit 3451f162d63c53b914237cf6ff47cb1ae7189812 on the main branch.

**Reply by MonkeyMatrix** [reply]
Looks good to me. I was unsure if I should capitalize colonies throughout the project too. Thanks!

**Reply by OcelotWalrus** [reply]
You're welcome! If you have any questions like this, just ask!

**Comment by OcelotWalrus** [comment]
> [Empire/empire prototypes outfits.txt](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/aa826ce410b70cbdf1acc909267705f234aa4e7e/data/Empire/empire%20prototypes%20outfits.txt#L254): How should this be rephrased?
>
>    It will also deal a noticeable amount of ion damage that can completely disable your ship energy long enough for you to disable them.

Hey, about that phrase, I think it should be rephrased like this:
_"It also deals a noticeable amount of ion damage to the target that can completely take down the target energy to 0."_

**Reply by OcelotWalrus** [reply]
Also about the Tre-Phasic and Tri-Phasic, I don't think it needs to be changed. Tre-Phasic is just fine.

**Reply by MonkeyMatrix** [reply]
Thanks for the rephrased sentence and clarification!

**Reply by OcelotWalrus** [reply]
Do you want me to change the phrase on the main branch?

**Reply by MonkeyMatrix** [reply]
Looking at it a second time, I think we should change it to, "It also deals a noticeable amount of ion damage to the target that can completely wipe out their energy" or "It also deals a noticeable amount of ion damage to the target that can completely wipe out its energy." (I prefer the first one) But, if it can take out the energy on practically any ship with one attack, then we could change the "a noticeable" amount to something like "an extreme," "an exceptional," or "an incredible" amount. Either way, I think it is fine to change it on the main branch.

**Reply by MonkeyMatrix** [reply]
> Also about the Tre-Phasic and Tri-Phasic, I don't think it needs to be changed. Tre-Phasic is just fine.

I wasn't sure if it was referring to the number three, such as the "tri" in tricycle. Thanks again!

**Reply by OcelotWalrus** [reply]
I think I prefer the first one too. I'm adding to the plugin right now.

**Comment by OcelotWalrus** [comment]
I also though that maybe this discussion/pull request should include revamping the spaceport/planet descriptions! Since most of them are outdated or just blank. For sure it would be an awesome addition.

But maybe it should be on another branch/pull request/discussion.

**Reply by MonkeyMatrix** [reply]
I will probably be going through this anyways :D. I was going through each file in the data folder one by one, starting with the Drak. I am currently on the empire starts.txt file. I took a break from doing this after spending a week or two going over the files in my free time. I will probably start again this week.

**Reply by OcelotWalrus** [reply]
Okay sure, take your time.

**Comment by MonkeyMatrix** [comment]
Should we say Mineria produced 4894B or 4.894T credits? (I prefer trillion, but I want to see what OcelotWalrus wants)

https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/14c5bbe02f07811e135aade264b1660756d767b7/data/Empire/empire%20starts.txt#L23

**Reply by OcelotWalrus** [reply]
trillion for sure.

**Comment by MonkeyMatrix** [comment]
Would each government have their own "app" for missions, trading, etc. inside that government? Or, would they all have agreed on a system (or perhaps by coincidence) in which they congregate these things into a universal listing?

https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/f4618b48c1ce66836054ac51abc63930e9e11272/data/Empire/empire%20storyline%200.txt#L78

**Reply by OcelotWalrus** [reply]
I think you miss understood. It's an app to get the special empire workforce missions from the storyline.

**Reply by MonkeyMatrix** [reply]
This cleared it up for me. Thanks!

**Reply by OcelotWalrus** [reply]
You're welcome!

**Comment by MonkeyMatrix** [comment]
Should we keep 1M or use 1,000,000 or one million in its place? I understand billions and trillions of credits in numerical form may be too much.

https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/f4618b48c1ce66836054ac51abc63930e9e11272/data/Empire/empire%20storyline%200.txt#L235

**Reply by lumbar527** [reply]
I think it would look best as `one million`.

**Reply by OcelotWalrus** [reply]
I prefer the 1M syntax because it make it stand out from the text.

**Reply by MonkeyMatrix** [reply]
I will keep 1M for now. Thanks for the feedback!

**Reply by OcelotWalrus** [reply]
You're welcome!

**Comment by MonkeyMatrix** [comment]
Are we unloading one or multiple Void Sprites here?

https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/blob/f4618b48c1ce66836054ac51abc63930e9e11272/data/Empire/empire%20storyline%201.txt#L200

**Reply by lumbar527** [reply]
One in the previous mission.
